roles/crawler/tasks/configure.yml

```diff
@@ -40,8 +40,11 @@
 
   - name: Populate config.ini.
     template:
-      src: config.ini.j2
-      dest: "{{ fpsd_crawler_project_directory }}/fpsd/config.ini"
+      src: fpsd.ini.j2
+      dest: "/etc/fpsd.ini"
+      owner: "{{ fpsd_crawler_system_account }}"
+      group: "{{ fpsd_crawler_system_account }}"
+      mode: "0770"
```

Comment:
I want all users in `fpsd` group to be able to modify the file. Hence `0660`. Ownership could be `root:fpsd` or `fpsd:fpsd`--wouldn't make a difference because the `fpsd` user is in `fpsd` group.